notebook/notebookapp.py

```diff
@@ -728,6 +729,15 @@ def _mathjax_url_changed(self, name, old, new):
         else:
             self.log.info("Using MathJax: %s", new)
 
+    mathjax_config = Unicode("", config=True,
+        help="""The MathJax.js configuration file that is to be used."""
+    )
+    def _mathjax_config_default(self):
```

Comment:
You don't need a dynamic default here, it can be in the constructor:

```python
mathjax_config = Unicode("TeX-AMS_HTML-full,Safe", config=True...
```